src/v/cluster/partition.cc

```diff
@@ -1134,61 +1134,29 @@ ss::future<> partition::unsafe_reset_remote_partition_manifest(iobuf buf) {
 
     // Add command to drop manifest. When applied, the current manifest
     // will be replaced with a default constructed one.
```

Comment:
nit: could we update this comment? now, the manifest gets replaced with the one from `req_m`